scripts/whats_left_modules.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# create directory in case it doesn't exist
+mkdir -p ../_data/whats_left
+ 
+# start a new csv file for modules
+echo "module" > ../_data/whats_left/modules.csv
+
+# read the temp file and look for "(entire module)"
+# save the output as csv 
+cat ../_data/whats_left.temp | grep "(entire module)" | cut -d ' ' -f 1 >> ../_data/whats_left/modules.csv
```

Comment:
This is the bash script that we can schedule with a github action. It gets the "entire modules" only.

Reply:
Should we invoke it in the action in RustPython/RustPython that creates `whats_left.temp`?

Reply:
The downside of invoking from `RustPython/RustPython` is that the action there will be even more complex and harder to maintain. If we need to add other scripts, we will keep having to do cross referencing.

We can try to trigger the script here by using on `push` + `path`. Here is how the action would look like

```
on:
  push:
    paths:
    - '**.temp'
``` 
Here is the docs on GitHub
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths 

This should work, do I give it a try?